xml/depl_nodes.xml

```diff
@@ -2467,7 +2457,7 @@ getent group postgres | cut -d: -f3</screen>
     </listitem>
    </varlistentry>
    <varlistentry>
-    <term><guimenu>User Name/Password for Accessing NetApp</guimenu>
```

Comment:
Ok I'll look at them again. I took a quick peek to verify that they were multiple fields, and didn't pay much attention to the field labels.